### PR TITLE
[WIP] Fixing prints to be Python 3.0 compatible

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 8bd516c900847330a8d7cda64671ab8fd2cf30f8:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8bd516c900847330a8d7cda64671ab8fd2cf30f8:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8bd516c900847330a8d7cda64671ab8fd2cf30f8:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8bd516c900847330a8d7cda64671ab8fd2cf30f8:

`artifactor/__init__.py`:
File lint OK :cake: :punch: :cookie:

`docs/inventory.py`:
File lint OK :cake: :punch: :cookie:

`scripts/apishow.py`:
File lint OK :cake: :punch: :cookie:

`scripts/cleanup_old_vms.py`:
File lint OK :cake: :punch: :cookie:

`scripts/cleanup_openstack_fips.py`:
File lint OK :cake: :punch: :cookie:

`scripts/enable_external_db.py`:
File lint OK :cake: :punch: :cookie:

`scripts/enable_internal_db.py`:
File lint OK :cake: :punch: :cookie:

`scripts/ntp_clock_set.py`:
File lint OK :cake: :punch: :cookie:

`scripts/template_tester.py`:
File lint OK :cake: :punch: :cookie:

`scripts/template_upload_rhevm.py`:
File lint OK :cake: :punch: :cookie:

`utils/tests/test_wait.py`:
File lint OK :cake: :punch: :cookie:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8bd516c900847330a8d7cda64671ab8fd2cf30f8:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7a4104e73edf928077ca0ac5e3b35db1f3edf53c:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 941f6594954bd10eb67340d6ce9b0bcf69d223a1:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 9dce71b4955f4a6e5597fae8f1086e71c963c345:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
There are some 'too many arguments to the format method' errors in landscape.

**Comment by psav** [comment]
There were, LS has not updated since my last push

**Comment by psav** [comment]
@mfalesni can you eagle eye this :)

File: scripts/template_upload_scvmm.py [hunk 1]
```diff
@@ -42,21 +42,21 @@ def check_template_exists(client, name):
 
 def upload_vhd(client, url, library, vhd):
 
-    print "SCVMM: DOwnloading VHD file, then updating Library"
+    print("SCVMM: DOwnloading VHD file, then updating Library")
 
-    script11 = "(New-Object System.Net.WebClient).DownloadFile( \
-        '" + url + "', '" + library + vhd + "');"
+    script11 = "(New-Object System.Net.WebClient).DownloadFile({}, {}{});".format(
```
Comment (mfalesni) [inline scripts/template_upload_scvmm.py]: You are missing quotes here if I see it correctly.

File: utils/tests/test_wait.py [hunk 2]
```diff
@@ -25,7 +25,7 @@ def test_simple_wait():
     ec, tc = wait_for(incman.i_sleep_a_lot,
                       fail_condition=0,
                       delay=.05)
-    print "Function output %s in time %s " % (ec, tc)
+    print("Function output %s in time %s " % (ec, tc))
```
Comment (mfalesni) [inline utils/tests/test_wait.py]: while you modified the formatters in the print statements, you can also do it in this file, this looks like it was forgotten :)

**Comment by dajoRH** [comment]
Lint report for commit 3073f6383c71201b199a5947467907a9fc07066e:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
Planting the bomb ...

**Pull request merged**